flixel/addons/effects/FlxGlitchSprite.hx

```diff
@@ -69,17 +86,12 @@ class FlxGlitchSprite extends FlxSprite
 		}
 		else
 		{
-			_time += elapsed;
+			_time += FlxG.elapsed;
```

Comment:
This PR is kind of obsolete with https://github.com/HaxeFlixel/flixel-addons/pull/205.